packages/suite/src/views/wallet/anonymize/components/CoinjoinSetupStrategies.tsx

```diff
@@ -72,25 +74,40 @@ interface CoinjoinSetupStrategiesProps {
 
 export const CoinjoinSetupStrategies = ({ account }: CoinjoinSetupStrategiesProps) => {
     const [strategy, setStrategy] = useState<CoinJoinStrategy>('recommended');
+    const [maxFee, setMaxFee] = useState(3);
+    const [skipRounds, setSkipRounds] = useState(true);
     const [connectedConfirmed, setConnectedConfirmed] = useState(false);
     const [termsConfirmed, setTermsConfirmed] = useState(false);
 
     const actions = useActions({
         startCoinjoinSession,
     });
+    const coordinatorData = useSelector(state => state.wallet.coinjoin.clients[account.symbol]);
+    const targetAnonymity = useSelector(selectCurrentTargetAnonymity);
+    const { notAnonymized } = useSelector(selectCurrentCoinjoinBalanceBreakdown);
+
+    if (!coordinatorData || targetAnonymity == null) {
+        return null;
+    }
 
     const isCustom = strategy === 'custom';
     const allChecked = connectedConfirmed && termsConfirmed;
-    const fee = (Number(account.balance) * 0.003).toString();
+    const fee = new BigNumber(notAnonymized).times(coordinatorData.coordinatorFeeRate).toString();
 
     const reset = () => setStrategy('recommended');
     const toggleConnectConfirmation = () => setConnectedConfirmed(current => !current);
     const toggleTermsConfirmation = () => setTermsConfirmed(current => !current);
     const anonymize = () =>
-        actions.startCoinjoinSession(
-            account,
-            COINJOIN_STRATEGIES[strategy === 'custom' ? 'recommended' : strategy], // TODO: enable custom strategy
-        );
+        actions.startCoinjoinSession(account, {
+            maxCoordinatorFeeRate: MAX_COORDINATOR_FEE,
```

Comment:
get coordinator fee rate from status (state.wallet.coinjoin.clients[network].coordinatorFeeRate)

Reply:
What does this value actually represent and what unit is it? I will add some comment explaining it. If, the `coordinatorFeeRate` is 0.003, how do I calculate `maxCoordinatorFeeRate `?

Reply:
Please check 93673b2e59367ed13545dc2e5117e9c1433d505d